main.go

```diff
@@ -54,6 +54,7 @@ func Router(Server *socketio.Server) *mux.Router {
 	r.HandleFunc("/posts/{post_id}/comments", blog.GetBlogComments).Methods("GET")
 	r.HandleFunc("/posts/{post_id}/comments", blog.CommentBlog).Methods("POST")
 	r.HandleFunc("/posts/search", blog.SearchBlog).Methods("GET")
+	r.HandleFunc("/posts/mail", blog.MailList).Methods("PATCH")
```

Comment:
Where is this function located? Abi it exists already?

Reply:
E dey blog.go file....

Reply:
https://github.com/zurichat/zc_core/pull/355/commits/7a47061d6e57e4aacce458671d6d9a411f2a3f91